Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse_description.md

```diff
@@ -0,0 +1,13 @@
+## Configure Cortex Xpanse
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Cortex Xpanse.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+     Server URL | The web UI with \`api-\` appended to front (e.g., https://api-xsiam.paloaltonetworks.com). For more information please see [get-started-with-cortex-xdr-apis](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-api/cortex-xdr-api-overview/get-started-with-cortex-xdr-apis). | True 
+     API Key ID | See [get-started-with-cortex-xdr-apis](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-api/cortex-xdr-api-overview/get-started-with-cortex-xdr-apis). | True 
+     API Key | See [get-started-with-cortex-xdr-apis](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-api/cortex-xdr-api-overview/get-started-with-cortex-xdr-apis). | True 
```

Comment:
The yml file does not have the API Key ID or API Key as parameters..
It has 
display: API Key ID
displaypassword: API Key
name: credentials